src/macro.jl

```diff
@@ -1128,7 +1128,7 @@ function make_many_actors(act!, args, ex1, outer::Vector, ex3, inner::Vector, ex
     end
     safeouter = setdiff(outer, unsafe)
 
-    if store.cuda > 0 && isdefined(store.mod, :KernelAbstractions)
+    if store.cuda === nothing || store.cuda > 0 && isdefined(store.mod, :KernelAbstractions)
         kernel = gensym(:🇨🇺)
```

Comment:
... in which case this should revert
```suggestion
    if store.cuda > 0 && isdefined(store.mod, :KernelAbstractions)
        kernel = gensym(:🇨🇺)
        workgroupsize = store.cuda==true ? nothing : store.cuda  # cuda=true means "use auto-tuning"
```

Reply:
Probably should be:
```suggestion
    if store.cuda > 0 && isdefined(store.mod, :KernelAbstractions)
        kernel = gensym(:🇨🇺)
        workgroupsize = store.cuda === true ? nothing : store.cuda  # cuda=true means "use auto-tuning"
```
since `1 == true`

Reply:
Is 1 a plausible setting here? If so, then perhaps this idea isn't a great one... 

I think `threads=1` and `verbose=1` are the same as `true`; for threads plausible other numbers are much larger (the smallest case worth dividing between threads)

Reply:
I thought `threads=1` just meant "use one thread"? `1` is definitely a valid `workgroupsize`, although maybe not the most sensible one. There might still be situations where you really want to test something without multiple threads though. I do think it should be fine to differentiate between 1 and `true` here.

Reply:
OK, let's see how it goes... 

Buildkite got the latest KA version, and seems happy.